flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -1193,4 +1194,70 @@ private Stream<TestSetSpec> arraySliceTestCases() {
                                 " No match found for function signature ARRAY_SLICE()")
                         .testSqlValidationError("ARRAY_SLICE(null)", "Illegal use of 'NULL'"));
     }
+
+    private Stream<TestSetSpec> splitTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.SPLIT)
+                        .onFieldsWithData("123,123,23", null, ",123,123", ",123,123,", 123, "12345")
```

Comment:
I would use `,123,,123,` instead of `,123,123,` so there is a test case for contiguous delimiters as well.

Reply:
Yes, you are right, I will add that test case, Thank you!